qkchash/qkchash.py

```diff
@@ -263,5 +261,18 @@ def test_qkchash_perf():
     print("Equal: ", h0 == h1[0 : len(h0)])
 
 
+def print_test_vector():
+    cache = make_cache(CACHE_ENTRIES, bytes())
+    print("Cache size: ", len(cache))
+    print("Hash of empty:")
+    h0 = qkchash(bytes(), bytes(), cache)
+    print(deserialize_hash(h0["mix digest"]))
```

Comment:
consider adding the result to unittest cases?

Reply:
Nice suggestion!